src/common/debug-requests.ts

```diff
@@ -29,7 +29,8 @@ export interface DebugRequestTypes {
 
 export interface DebugEvents {
     'memory': DebugProtocol.MemoryEvent,
-    'stopped': DebugProtocol.StoppedEvent
+    'stopped': DebugProtocol.StoppedEvent,
+    'output': DebugProtocol.OutputEvent,
```

Comment:
Is this used?

Reply:
No, it was accidentally left over from prior work now unused. I've removed it.